bitbox-bridge/release/Dockerfile

```diff
@@ -9,7 +9,9 @@ RUN apt-get update -y && apt-get install -y --no-install-recommends \
     cmake \
     curl \
     wget \
-    software-properties-common 
+    software-properties-common \
+    apt-transport-https \
+    ca-certificates
```

Comment:
I would expect ca-certificates to be installed as a dependency on apt-transport-https:

![image](https://user-images.githubusercontent.com/25405/111618019-2a7c1400-87e4-11eb-8712-37dffc361e12.png)

https://packages.ubuntu.com/xenial/apt-transport-https

Reply:
Does `recommends` means it is gonna be installed?

Reply:
Docs seem unclear, but based on the wording I'd say better add explicitly.

Reply:
:+1: